nodejs/egg-pterodactyl-node-js-generic.json

```
@@ -10,16 +10,17 @@
     "description": "a generic node.js egg\r\n\r\nThis will clone a git repo. it defaults to master if no branch is specified.\r\n\r\nInstalls the node_modules on install. If you set user_upload then I assume you know what you are doing.",
     "features": null,
     "docker_images": {
-        "Nodejs 12": "ghcr.io/parkervcp/yolks:nodejs_12",
-        "Nodejs 14": "ghcr.io/parkervcp/yolks:nodejs_14",
-        "Nodejs 16": "ghcr.io/parkervcp/yolks:nodejs_16",
-        "Nodejs 17": "ghcr.io/parkervcp/yolks:nodejs_17",
-        "Nodejs 18": "ghcr.io/parkervcp/yolks:nodejs_18",
-        "Nodejs 19": "ghcr.io/parkervcp/yolks:nodejs_19",
-        "Nodejs 20": "ghcr.io/parkervcp/yolks:nodejs_20",
-        "Nodejs 21": "ghcr.io/parkervcp/yolks:nodejs_21",
-        "Nodejs 22": "ghcr.io/parkervcp/yolks:nodejs_22",
-        "Nodejs 23": "ghcr.io/parkervcp/yolks:nodejs_23"
+        "Nodejs 24": "ghcr.io\/parkervcp\/yolks:nodejs_24",
+        "Nodejs 23": "ghcr.io\/parkervcp\/yolks:nodejs_23",
+        "Nodejs 22": "ghcr.io\/parkervcp\/yolks:nodejs_22",
+        "Nodejs 21": "ghcr.io\/parkervcp\/yolks:nodejs_21",
+        "Nodejs 20": "ghcr.io\/parkervcp\/yolks:nodejs_20",
+        "Nodejs 19": "ghcr.io\/parkervcp\/yolks:nodejs_19",
+        "Nodejs 18": "ghcr.io\/parkervcp\/yolks:nodejs_18",
+        "Nodejs 17": "ghcr.io\/parkervcp\/yolks:nodejs_17",
+        "Nodejs 16": "ghcr.io\/parkervcp\/yolks:nodejs_16",
+        "Nodejs 14": "ghcr.io\/parkervcp\/yolks:nodejs_14",
+        "Nodejs 12": "ghcr.io\/parkervcp\/yolks:nodejs_12"
```

```suggestion
        "Nodejs 12": "ghcr.io/parkervcp/yolks:nodejs_12",
        "Nodejs 14": "ghcr.io/parkervcp/yolks:nodejs_14",
        "Nodejs 16": "ghcr.io/parkervcp/yolks:nodejs_16",
        "Nodejs 17": "ghcr.io/parkervcp/yolks:nodejs_17",
        "Nodejs 18": "ghcr.io/parkervcp/yolks:nodejs_18",
        "Nodejs 19": "ghcr.io/parkervcp/yolks:nodejs_19",
        "Nodejs 20": "ghcr.io/parkervcp/yolks:nodejs_20",
        "Nodejs 21": "ghcr.io/parkervcp/yolks:nodejs_21",
        "Nodejs 22": "ghcr.io/parkervcp/yolks:nodejs_22",
        "Nodejs 23": "ghcr.io/parkervcp/yolks:nodejs_23",
        "Nodejs 24": "ghcr.io/parkervcp/yolks:nodejs_24"
```
Pterodactyl sorts them differently